### ISSUE TITLE
Unhandled Promise rejection

### ISSUE THREAD

**Issue by omitron** [timostamm/protobuf-ts#86]
We face an error with an unhandled promise after aborting a stream.
It seams that readGrpcWebResponseBody(...) should have a return something.

https://github.com/timostamm/protobuf-ts/blob/b2b38872abe9d379645f92b81750b3807c2f35b0/packages/grpcweb-transport/src/grpc-web-format.ts#L137

What do you think? Any recomondations?

**Comment by timostamm** [comment]
I have also seen warnings about unhandled promise rejections.  But when I was ready to investigate, I could no longer reproduce the warning. It is possible that the issue has been fixed in issue #81. 

The fix of #81 is not released yet. So it may be possible that the warning goes away with the next release. 

I am going to make a release now. I suggest you try it again with the new release. If the warning persists, it would be very helpful to have a reproducible example, then I can fix the root cause.

**Comment by timostamm** [comment]
v.2.0.0-alpha-13 has just been released in the "next" channel. You can update with `@protobuf-ts/runtime-rpc@next`. Please make sure to update all @protobuf-ts packages to the new version and re-compile your .proto files.

**Comment by omitron** [comment]
Thanks for the fast response. Unfortunately, it didn't do the trick. The problem still exists (Unhandled Promise rejection: The user aborted a request. ; Zone: <root> ; Task: Promise.then ; Value: RpcError: The user aborted a request.).

We use an  AbortController to stop a stream from client side on certain conditions (e.g. user request). When we do so, the stream is stopped but we get the error message. So, the problem is basically only the error message not the behavior.

**Comment by timostamm** [comment]
I did a little more digging and was able to repro with a server streaming method, grpc-web and Google Chrome. I have added a workaround that fixes the issue. 

I also noticed that the "code" property of the raised `RpcError` was "INTERNAL" for a cancelled server streaming method. It should be "CANCELLED". This is also fixed. 

Released in v.2.0.0-alpha-15. Let me know if this works for you, and thanks for the feedback.
